magic-square/test/magic_square/puzzle_test.clj

```diff
@@ -5,6 +5,8 @@
 (defn sum-rows [m]
   (map #(reduce + %) m))
 
+(def values [1.0 1.5 2.0 2.5 3.0 3.5 4.0 4.5 5.0])
```

Comment:
Generally, for all of these exercise, you don't need to change the test. You only should change the code (in the src/ folder) so that tests work